src/socket.c

```diff
@@ -54,22 +61,29 @@
 	} \
 } while(0)
 
-#define PTHREADS_SOCKET_ERROR() do { \
-	int eno = php_socket_errno(); \
-	char *estr = eno != SUCCESS ? \
-		php_socket_strerror(eno, NULL, 0) : \
-		NULL; \
-	zend_throw_exception_ex(spl_ce_RuntimeException, eno, \
-		"Error (%d): %s", eno, estr ? estr : "unknown"); \
-	if (eno != SUCCESS) { \
-		if (estr) { \
-			efree(estr); \
+#define PTHREADS_HANDLE_SOCKET_ERROR(eno) do { \
```

Comment:
Conventionally, parentheses should be placed around `eno` in the macro body.

Reply:
Parentheses added, should be fixed